docs/PacketFence_OpenWrt-Hostapd_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,152 @@
+OpenWrt BarrierBreaker 14.07 with hostapd Quick Integration Guide
+=================================================================
+:encoding: UTF-8
+:lang: en
+:doctype: book
+
+include::includes/global-attributes.asciidoc[]
+
+About this Guide
+----------------
+This guide has been created in order to help sales engineers, product managers, or network specialists demonstrate the PacketFence capabilities on-site with an existing or potential customer.  It can also provide guidelines to setup a proof of concept for a potential PacketFence deployment using OpenWrt BarrierBreaker 14.07 with Hostapd.
+
+Introduction
+------------
+This guide will provide an example for the configuration of an Open SSID (not crypted) and a Secure SSID (802.1x). You will to install wpad and hostapd. This two SSIDs will do radius authentication against PacketFence.
```

Comment:
an Open -> an open
not crypted -> not encrypted
a Secure -> a secured
This two -> These two
radius -> RADIUS